scalatags/src/scalatags/DataTypes.scala

```diff
@@ -32,58 +32,58 @@ trait DataConverters{
      *
      * MDN
      */
-    def px = x + "px"
+    def px = x.toString + "px"
```

Comment:
Imo this would look a little nicer as
```scala
def px = s"${x}px"
```
Just personal opinion though.

Reply:
Yes, agree!